roles/shared_storage/tasks/main.yml

```diff
@@ -22,7 +22,7 @@
 
 - name: 'Create "home" Logical File System (LFS) on Physical File Systems (PFSs) mounted on SAIs.'
   file:
-    path: "/mnt/{{ item.pfs }}/{{ item.lfs }}"
+    path: "/mnt/{{ item.pfs | regex_replace('/.+$', '') }}/{{ item.lfs }}"
```

Comment:
could you please change this into something like {{ item.pfs.split("/")[1:]|join("/") }}
(stolen from here https://stackoverflow.com/questions/35271368/get-parent-directory-in-ansible)
I do not feel confident that i could debug this, should the need arise.

Reply:
That won't work as the value of the ```pfs``` key may contain an extra part after a ```/``` that has to be removed in certain cases, but this is only to support the data handling storage systems. All others don't have slash forward and subfolder appended and trying to split and use the first value using ```.split('/')[1]``` errors with
```
The error was: list object has no element 1
```
The regex handles this gracefully: it removes it when present and does not complain when there is nothing to remove.